plugins/inlinetoolbar/plugin.js

```diff
@@ -25,10 +30,19 @@
 		this.listeners = [];
 	};
 
+	var stylesLoaded = false;
+
 	CKEDITOR.plugins.add( 'inlinetoolbar', {
 		requires: 'balloonpanel',
 		init: function() {
+			if ( !stylesLoaded ) {
+				CKEDITOR.document.appendStyleSheet( this.path + 'skins/' + CKEDITOR.skinName + '/inlinetoolbar.css' );
+				stylesLoaded = true;
+			}
 			CKEDITOR.ui.inlineToolbarView.prototype = CKEDITOR.tools.extend( {}, CKEDITOR.ui.balloonPanel.prototype );
+			if ( CKEDITOR.ui.inlineToolbarView.prototype.templateDefinitions && CKEDITOR.ui.inlineToolbarView.prototype.templateDefinitions.panel.indexOf( 'cke_inlinetoolbar' ) === -1 ) {
```

Comment:
This is not an elegant way to add this class.

You could just add `this.parts.panel.addClass( 'cke_inlinetoolbar' );` in `build` method and that will do it.

I have pushed unit test checking if the class is added by the plugin.

Reply:
I think it is because`CKEDITOR.ui.inlineToolbarView.prototype.templateDefinitions.panel` is still a string there. So that's why I suggested adding comment there at it is a little misleading. But maybe there is a better way to handle this, WDYT @mlewand ?

Reply:
I mean the solution that I have proposed above is much more readable and elegant.

String replacement looks bad and makes a very imprecise replacement that could as well replace any other part of HTML, say someone will add another class `cke_balloonpanel`  `cke_balloon_custom` or an attribute that will include it.

Reply:
I will go along with @mlewand solution with adding class in build method.